src/components/button/Button.scss

```diff
@@ -0,0 +1,23 @@
+.button {
+  justify-self: center;
+  height: 40px;
+  border-width: 0px;
+  padding-inline: 30px;
+  color: orange;
+  font-weight: bold;
+
+  &:hover {
+    color: white;
+  }
+
+  &--primary {
+    border-radius: 25px;
+    background-color: white;
+    background-repeat: repeat;

```

Comment:
The `background-repeat` property is used for images, I don't think that you need it.

Could you try to remove it and see if everything is looking good?